apps/web/src/components/Basenames/RegistrationValueProp/index.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import faceScan from './assets/faceScan.svg';
+import currencies from './assets/currencies.svg';
+import sofort from './assets/sofort.svg';
+import globe from './assets/globeWhite.webm';
+import classNames from 'classnames';
+import ImageAdaptive from 'apps/web/src/components/ImageAdaptive';
+import { StaticImageData } from 'next/image';
+
+type ValuePropProps = {
+    title: string;
+    description: string;
+    image: StaticImageData;
+    color: string;
+};
+
+const valueClass = classNames(
+    'relative px-2 py-2',
+  );
+
+function ValueProp({ title, description, image, color }: ValuePropProps) {
+    return (
+        <div className="rounded-xl flex items-center bg-gray-5 py-2 px-2 space-x-2 shadow-md border border-palette-line/20 bg-opacity-80">
+            {/* bg-[#0052FF],blue --- bg-[#22AD73],green --- bg-[#8A55E9], purple */}
+            <div className={`bg-[${color}] rounded-2xl px-6 py-3`}>
```

Comment:
double check that this color is working

Reply:
same thing here, I'd just pass a bgClassName to the `ValueProp`.

Then you can just pass `<ValueProp bgClassName="bg-[#0f0f0]"...` and you won't have to do the commented hack.